nomos-services/time/Cargo.toml

```diff
@@ -22,8 +22,9 @@ tokio              = { version = "1", features = ["macros", "sync"] }
 tokio-stream       = "0.1"
 
 [features]
-ntp   = ["dep:sntpc", "dep:thiserror"]
-serde = ["cfg_eval", "dep:nomos-utils", "dep:serde", "dep:serde_with"]
+ntp       = ["dep:sntpc", "dep:thiserror"]
+serde     = ["cfg_eval", "dep:nomos-utils", "dep:serde", "dep:serde_with"]
```

Comment:
I know it's unrelated, but would you add the `dep:` prefix to this dependency, so we avoid exposing the `cfg_eval` feature, which I guess we don't want to?